build-tools/scripts/Configuration.OperatingSystem.Windows.props

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Given that we now have `build-tools/scripts/dependencies/linux-*` files, this should probably be named `build-tools/scripts/dependencies/Windows-Configuration.OperatingSystem.props`.